code-style/git/README.md

```diff
@@ -322,8 +335,11 @@ Voila! You're done. You've successfully rebased a branch onto the master branch!
 [Draft PRs]: https://github.blog/2019-02-14-introducing-draft-pull-requests/
```

Comment:
Since the text for this changed, we'd have to change the link here too.
```suggestion
[Draft PR]: https://github.blog/2019-02-14-introducing-draft-pull-requests/
```